ui/packages/consul-ui/app/templates/dc/show/serverstatus.hbs

```diff
@@ -0,0 +1,240 @@
+          <notice.Header>
+            <strong>Error!</strong>
+          </notice.Header>
+          <notice.Body>
+            <p>
+              You no longer have access to this service
+            </p>
+          </notice.Body>
+        </Notice>
+      {{else}}
+        <Notice
+          {{notification
+            sticky=true
+          }}
+          class="notification-update"
+          @type="warning"
+        as |notice|>
+          <notice.Header>
+            <strong>Warning!</strong>
+          </notice.Header>
+          <notice.Body>
+            <p>
+              An error was returned whilst loading this data, refresh to try again.
+            </p>
+          </notice.Body>
+        </Notice>
+      {{/if}}
+    </BlockSlot>
+
+    <BlockSlot @name="loaded">
+      <div class="tab-section">
+
+          <section
+            class={{class-map
+              'server-failure-tolerance'
+            }}
+          >
+
+            <header>
+              <h2>
+                {{compute (fn route.t 'tolerance.header')}}
+              </h2>
+            </header>
+
+            <section
+              class={{class-map
+                (array 'immediate-tolerance')
+              }}
+            >
+              <header>
+                <h3>
+                  {{compute (fn route.t 'tolerance.immediate.header')}}
+                </h3>
+              </header>
+              <dl
+                class={{class-map
+                  (array 'warning' (and
+                    (eq item.FailureTolerance 0)
+                    (eq item.OptimisticFailureTolerance 0)
+                  ))
+                }}
+              >
+                <dt>
+                  {{compute (fn route.t 'tolerance.immediate.body')}}
+                </dt>
+                <dd>
+                  {{item.FailureTolerance}}
+                </dd>
+              </dl>
+            </section>
+
+            <section
+              class={{class-map
+                (array 'optimistic-tolerance')
+              }}
+            >
+              <header>
+                <h3>
+                  {{compute (fn route.t 'tolerance.optimistic.header')}}
+              {{#if (not (can 'read zones'))}}
+                  <em>
+                    {{t 'common.ui.enterprisefeature'}}
+                  </em>
+              {{/if}}
+                  <span
+                    {{tooltip 'With > 30 seconds between server failures, Consul can restore the Immediate Fault Tolerance by replacing failed active voters with healthy back-up voters when using redundancy zones.'}}
+                  >
+                  </span>
+                </h3>
+              </header>
+              <dl
+                class={{class-map
+                  (array 'warning' (eq item.OptimisticFailureTolerance 0))
+                }}
+              >
+                <dt>
+                  {{compute (fn route.t 'tolerance.optimistic.body')}}
+                </dt>
+                <dd>
```

Comment:
Code looks ok... question, should the orange info icon have a tooltip like the grey one to the right of "Optimistic" or is that expected? Was taking a look at the UI to better understand what was going on

<img width="1051" alt="image" src="https://user-images.githubusercontent.com/1372946/159829235-b9f8505f-3b6b-4192-bb74-beaf353e4ed8.png">

Reply:
Hey @amyrlam 👋 , good question! Let me double check the designs, I think I remember seeing a screen with all the tooltips open for the entire page. There are quite a few to add still. What would be best from your perspective, add them all here or in this PR or add those a little later on?

Reply:
@johncowen Ah gotcha. TBH I don't 100% grok how on the OSS self-hosted side, you all manage what's in "main" vs when it's deployed. Imagine it's a bit different than a cloud product. So the code is fine with me as-is, just didn't know if that was an error in the UI